commands.js

```diff
@@ -272,6 +272,26 @@ exports.commands = {
 	},
 	backhelp: ["/back - Unblocks challenges and/or private messages, if either are blocked."],
 
+	rank: function (target, room, user) {
+		if (!target) target = user.name;
+
+		let buffer = '<div class="ladder"><table>';
+		buffer += '<tr><td colspan="8">User: <strong>' + Tools.escapeHTML(target) + '</strong></td></tr>';
```

Comment:
Construct the buffer inside the `then` function.

Long-term buffers are a huge waste of RAM if you don't actually need them.